### PR TITLE
information_schema.tables.version is experimental

### LINKED ISSUE #4479: Information Schema Doc Update
Re: [Information Schema](https://cockroachlabs.com/docs//v2.1/information-schema.html)

## Issue Description

The column `information_schema.tables.version` is an experimental CockroachDB extension and should be marked as such.

## Suggested Resolution 

Add a note "This column is an external feature used for internal purposes inside CockroachDB and its definition is subject to change without notice."

### PR DESCRIPTION
**Pull request by rmloveland**
Fixes #4479.

### CODE REVIEW

**Comment by rmloveland** [comment]
Direct link to changed section is [here](http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/e6d58996e3c55277e5f3f9929387078d490c66eb/dev/information-schema.html#tables).

File: v19.1/information-schema.md [hunk 1]
```diff
@@ -278,7 +278,7 @@ Column | Description
 `table_schema` | Name of the schema that contains the table.
 `table_name` | Name of the table.
 `table_type` | Type of the table: `BASE TABLE` for a normal table, `VIEW` for a view, or `SYSTEM VIEW` for a view created by CockroachDB.
-`version` | Version number of the table; versions begin at 1 and are incremented each time an `ALTER TABLE` statement is issued on the table.
+`version` | Version number of the table; versions begin at 1 and are incremented each time an `ALTER TABLE` statement is issued on the table. Note that this column is an external feature used for internal purposes inside CockroachDB and its definition is subject to change without notice.
```
Comment (knz) [inline v19.1/information-schema.md]: ... an experimental feature ...

File: v2.1/information-schema.md [hunk 2]
```diff
@@ -278,7 +278,7 @@ Column | Description
 `table_schema` | Name of the schema that contains the table.
 `table_name` | Name of the table.
 `table_type` | Type of the table: `BASE TABLE` for a normal table, `VIEW` for a view, or `SYSTEM VIEW` for a view created by CockroachDB.
-`version` | Version number of the table; versions begin at 1 and are incremented each time an `ALTER TABLE` statement is issued on the table.
+`version` | Version number of the table; versions begin at 1 and are incremented each time an `ALTER TABLE` statement is issued on the table. Note that this column is an external feature used for internal purposes inside CockroachDB and its definition is subject to change without notice.
```
Comment (knz) [inline v2.1/information-schema.md]: ditto

File: v19.1/information-schema.md (hunk 1 above)
Comment (rmloveland) [reply v19.1/information-schema.md]: Fixed.

File: v2.1/information-schema.md (hunk 2 above)
Comment (rmloveland) [reply v2.1/information-schema.md]: Fixed.

**Pull request merged**